src/PlatAPI.ts

```diff
@@ -323,7 +323,8 @@ export class PlatAPI {
             const inputSourcePaths = requirement.sources ?? [
                 ["request", "params", paramName],
                 ["request", "query", paramName],
-                ["request", "body", paramName]
+                ["request", "body", paramName],
+                ["request", "headers", paramName.toLowerCase()]
```

Comment:
I understand this block of code to essentially be the default path finding if there is none provided for some reason?

I added the headers path in as well even though I technically didn't need that for this particular fix.